guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,325 @@
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Lightning-address-mobile@2x.png"
+   alt-text = "Graphic depicting examples of lightning addresses"
+   width = 800
+   height = 348
+%}
+
+### Receive payments offline
+
+Non-custodial lightning wallets need to be online to receive payments. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}) in particular are regularly offline so receiving payments can be difficult.
+
+LWS can act as third parties that non-custodially hold onto a payment and forward it to the user once they come online. A product that offers this for users is [Greenlight](https://blockstream.com/lightning/greenlight/).
+
+This service is usually required to be offered alongside an LSP. Read more about this [here](https://medium.com/breez-technology/get-ready-for-a-fresh-breez-multiple-apps-one-node-optimal-ux-519c4daf2536).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Offline-payment.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile@2x.png"
+   alt-text = "Graphic depicting how an LWS helps users accept payments whilst offline"
+   width = 800
+   height = 348
+%}
+
+### Swaps
+
+Moving bitcoin between lightning and on-chain can be costly and complicated if done by opening and closing payment channels.
+
+LWS can offer [submarine swaps](https://blog.muun.com/a-closer-look-at-submarine-swaps-in-the-lightning-network/) to allow users to easily move bitcoin between lightning and on-chain without having to manage and deal with payment channels.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile@2x.png"
+   alt-text = "Graphic depicting how an LWS can conduct swaps moving bitcoin between on-chain and lightning"
+   width = 800
+   height = 348
+%}
+
+### Watchtowers
+
+Users need to regularly be online to monitor their payment channels to prevent cheating attempts. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}) in particular are regularly offline and so cheating attempts are more likely.
+
+A [watchtower](https://bitcoinops.org/en/topics/watchtowers/) LWS service monitors payment channels of users who are offline and can punish their counterparties if they attempt to steal the users funds.
```

Comment:
```suggestion
A lightning wallet must be online on a regular basis to track it's payment channels. A [daily spending wallet]({{ '/guide/daily-spending-wallet/' | relative_url }}), in particular, is offline whenever the user stops using the app. This means there is a greater risk of cheating attempts.

A [watchtower](https://bitcoinops.org/en/topics/watchtowers/) service can fix this. Watchtowers monitor the payment channels of offline users. If a counterparty attempts to steal a user's funds, the watchtower can step in to help. The watchtower can prevent the theft by submitting a justice transaction.
```

A couple of things.
- In sentence 1, we talk about a "user" being offline, but in sentence 2 we talk about a "wallet" being offline. Let's keep it consistent throughout the paragraph.
  - IMO, better to talk about the wallet being offline.
  - Example: I don't really care if the user is looking at the app or not -- if Apple and Google decided that bitcoin wallets could conduct lightning activity in the background, that would basically solve a lot of these issues. So the issue itself is not necessarily about the user being online but rather the node (the app) being online.
- Changed "punishment" to "justice transaction" in futile attempt to thwart BDSM jokes before they arrive.
- Simplified language a good bit
- Do we need to end words like "watchtower" with "LWS"? I feel like the words are more easily understood if we strip out jargon. Already have to teach the reader what a watchtower is, calling it a "watchtower LWS service" just makes it sound more complicated.